docs/swagger.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
Can you add something to the pull request description (or `README.md`) about the Swagger API definitions you added here?